nova/tests/unit/network/test_neutronv2.py

```diff
@@ -2004,10 +1996,7 @@ def test_validate_networks_no_quota(self):
                 {'networks': self.nets2})
         self.moxed_client.show_quota(
             uuids.my_tenant).AndReturn(
-                    {'quota': {'port': 2}})
```

Comment:
Why did we have to change quota here? I thought it's only about not calling `list_ports()` anymore.